project/src/components/offer/offer.jsx

```diff
@@ -49,7 +57,11 @@ function Offer({
             <b className="place-card__price-value">&euro;{price}</b>
             <span className="place-card__price-text">&#47;&nbsp;night</span>
           </div>
-          <button className={`place-card__bookmark-button ${offer.isFavorite ? 'place-card__bookmark-button--active' : ''} button`} type="button">
+          <button
+            className={`place-card__bookmark-button ${offer.isFavorite ? 'place-card__bookmark-button--active' : ''} button`}
+            type="button"
+            onClick={() => handleFavoriteClick(offer.isFavorite ? 0 : 1)}
```

Comment:
Давай так:
```suggestion
            onClick={handleFavoriteClick}
```
Зачем две функции, если можно обойтись одной)

Reply:
Сделано